Doc/tutorial/inputoutput.rst

```diff
@@ -172,7 +172,7 @@ Positional and keyword arguments can be arbitrarily combined::
 If you have a really long format string that you don't want to split up, it
 would be nice if you could reference the variables to be formatted by name
 instead of by position.  This can be done by simply passing the dict and using
-square brackets ``'[]'`` to access the keys ::
+square brackets ``'[]'`` to access the keys. ::
```

Comment:
More or less. Periods before an example always ends with a "." or with a ":". Right?
I chose the first one for no special reason.

Reply:
> More or less. Periods before an example always ends with a "." or with a ":". Right?

Here are all of the correct ways to use examples (as far as I'm aware):

```
This can be done by simply passing the dict and using square brackets ``'[]'``
to access the keys::
```

```
This can be done by simply passing the dict and using square brackets ``'[]'``
to access the keys. For example::
```

```
This can be done by simply passing the dict and using square brackets ``'[]'``
to access the keys. ::
```

Those types of formatting corrections aren't worth their own PR, but it can be useful to touch up on them if you're working on a nearby area for something more significant (such as with this PR). The quote used in PEP 8, "A foolish consistency is the hobgoblin of little minds", would be applicable to changing a section *exclusively* to use the correct formatting conventions.